src/item.cpp

```diff
@@ -2237,14 +2268,20 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
-    if( t == nullptr ) {
+    float pockets = 1;
+    auto t = find_armor_data();
+    if( t == nullptr )
         return 0;
-    }
+
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
-}
+    int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
 
+    if (item::item_tags.count("pocketed") > 0){
+        pockets = (volume() * get_coverage()) / 100;
+        return result + pockets;
+        }
```

Comment:
The else goes on the same line as the preceeding }
The code after the else must have brackets around it {}
The bracket after the else goes on the same line as the else.
You need to fix this in a lot of places.